contracts/script/README.md

```diff
@@ -0,0 +1,142 @@
+# Deploying Upgradeable Smart Contracts
+
+Upgradeable Smart contracts are deployed with Openzeppelin Defender to enable a deployment strategy that is more secure
+and also uses a multi-sig Safe wallet. When deploying using openzeppelin the `defender` profile in the `foundry.toml`
+file is used.
+
+Note: One must set the environment variable `FOUNDRY_PROFILE` accordingly to use the appropriate profile in the
+`foundry.toml` settings. E.g.
+
+```bash
+export FOUNDRY_PROFILE=defender
+```
+
+The profile, `profile.default` is used by default. Setting the `FOUNDRY_PROFILE` variable overrides the `foundry.toml`
+settings.
+
+## Prerequisites
+
+1. Create a multisig wallet using [Safe](https://app.safe.global/welcome/accounts) on the network you'd like to deploy
+   to.
+2. In OpenZeppelin Defender, create an Approval Process that requires the multisig wallet you created above.
```

Comment:
I get this message when using the address from step 1 in defender.


![image](https://github.com/EspressoSystems/espresso-sequencer/assets/1040871/d59d6e0b-7598-40e6-9e9c-b3d0653d06fd)